<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>676</width>
    <height>601</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Bermudo's Student Information System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="studentTab">
       <attribute name="title">
        <string>Students</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QFrame" name="frame">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="4">
                  <widget class="QPushButton" name="showButton">
                   <property name="minimumSize">
                    <size>
                     <width>172</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>172</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show / Refresh List</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="searchButton">
                   <property name="minimumSize">
                    <size>
                     <width>168</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>168</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>Search</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="searchBar">
                   <property name="minimumSize">
                    <size>
                     <width>113</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>113</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                    </font>
                   </property>
                   <property name="toolTip">
                    <string>Search by ID Number e.g. (2020 - 1971)</string>
                   </property>
                   <property name="toolTipDuration">
                    <number>10</number>
                   </property>
                   <property name="statusTip">
                    <string notr="true">Search by ID Number e.g. (2020 - 1971)</string>
                   </property>
                   <property name="placeholderText">
                    <string>Search by ID Number</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="5">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="3">
                  <widget class="QPushButton" name="addButton">
                   <property name="minimumSize">
                    <size>
                     <width>169</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>169</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>Add Student</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="columnCount">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="coursesTab">
       <attribute name="title">
        <string>Courses</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="2">
                  <widget class="QPushButton" name="searchButton2">
                   <property name="minimumSize">
                    <size>
                     <width>168</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>168</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>Search</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QPushButton" name="showButton2">
                   <property name="minimumSize">
                    <size>
                     <width>172</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>172</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show / Refresh List</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QPushButton" name="addButton2">
                   <property name="minimumSize">
                    <size>
                     <width>169</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>169</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>Add Course</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="searchBar2">
                   <property name="minimumSize">
                    <size>
                     <width>113</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>113</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                    </font>
                   </property>
                   <property name="toolTip">
                    <string>Search by ID Number e.g. (2020 - 1971)</string>
                   </property>
                   <property name="toolTipDuration">
                    <number>10</number>
                   </property>
                   <property name="statusTip">
                    <string notr="true">Search by ID Number e.g. (2020 - 1971)</string>
                   </property>
                   <property name="placeholderText">
                    <string>Search by Course Code</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="5">
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget2">
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="columnCount">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
